<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>18</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NAMA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>10</y>
      <width>351</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>46</y>
      <width>351</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>54</y>
      <width>70</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>JURUSAN</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>440</width>
      <height>192</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <column>
     <property name="text">
      <string>Nama</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Jurusan</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_simpan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SIMPAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_edit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hapus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>HAPUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_batal">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>BATAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_loadData">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOAD DATA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
